src/caffeinated_whale_cli/commands/status.py

```
@@ -0,0 +1,89 @@
+        None,
+    )
+    if not frappe_container:
+        typer.echo("offline")
+        raise typer.Exit(code=0)
+
+    # Refresh status info
+    try:
+        frappe_container.reload()
+    except Exception:
+        typer.echo("offline")
+        raise typer.Exit(code=0)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)